community-roles.md

```diff
@@ -1,49 +1,46 @@
+
+## Repositories Manager
+The Repositories Manager controls Github, NPM, and other Solid organisation properties related to the storage, management, and distribution of official Solid projects. They are responsible for keeping these properties clean and well-organised on behalf of the Solid community. The Repositories Manager has admin rights of Solid tools must attend the weekly recurring community support meeting.
 
 ## Solid Core Contributor
-Solid Core Contributors work across projects on initiatives that enrich the
-Solid ecosystem as a whole. They must attend regular stand-ups and strategy
-meetings.
+Solid Core Contributors work across projects on initiatives that enrich the Solid ecosystem as a whole. Solid Core Contributors must attend the weekly recurring community support meeting.
 
 ## Project Core Contributor
-A Project Core Contributor is a key team member on an official Solid project.
-They may have issues assigned to them, review and merge pull requests, and must
-attend regular stand-ups and project strategy meetings.
-
-## Project Contributor
-A Project Contributor is anyone who contributes to an official Solid project.
-Contributions come in many forms, from coding and qa to experience design and
-technical content writing.
+A Project Core Contributor is a key team member on an official Solid project. Project Core Contributors must attend the weekly recurring community support meeting.
 
 ## Project Release Manager
-The Project Release Manager is responsible for determining which features and/or
-bug-fixes will make a given project release. They must attend regular stand-ups
-and project strategy meetings.
+The Project Release Managers are responsible for determining which features and/or bug-fixes will be merged into the release of their respective projects. Project release managers must attend the weekly recurring community support meeting.
+
+## Solid Coordinator
+Panel Coordinators are responsible for designing and running tests with the Solid Panelists as well as publishing test results to the Solid community. Panel Coordinators are expected to join the weekly user testing meeting which is moderated by the Community Manager.
+
+## Solid Panelist
+The Solid panel engages in continuous tests to improve the user experience. When a test is set up by the Solid panel coordinators the community manager will reach out to the Solid panel to ask if they would like to participate in this test. Tests will take approximately half an hour each on average. If the user tester is available and willing to take the test, they will be sent instructions by the Solid panel coordinators. Solid panelist can withdraw from the panel at any point. To volunteer to become a Solid panelist contact the community manager.
+
```

Comment:
We should be trying to get as much feedback from as many people as we can, and I worry that requiring them to assume a named role in the community just to try something and provide feedback is heavy-handed. I think having the role is OK, but I'm not sure that we should manage "membership" ranks associated with the role. Instead, possibly provide attribution of who was participated. 

This description is also quite skewed only to UX, when we would expect that panelists could provide feedback and discussion on other topics, from developer experience and linked data conventions to security.